### PR TITLE
Remove IsTest=true from some utility projects

### PR DESCRIPTION
**Pull request by mconnew**
This is blocking updating of versions pulled in by Aracade

### CODE REVIEW

**Comment by HongGit** [comment]
@mconnew why remove one, the other just change value from true to false?  why not remove both?

File: src/System.Private.ServiceModel/tests/Common/Infrastructure/Infrastructure.Common.csproj [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <OutputType>Library</OutputType>
     <CLSCompliant>false</CLSCompliant>
-    <IsTestProject>true</IsTestProject>
+    <IsPackable>false</IsPackable>
```
Comment (HongGit) [inline src/System.Private.ServiceModel/tests/Common/Infrastructure/Infrastructure.Common.csproj:6]: Why not remove this line as well for consistency?

**Comment by mconnew** [comment]
I think you might have misread the diff. I did remove it from both. I also added IsPackable=false to the Infrastructure project as it was in the unit test project. I wanted to make sure that removing the IsTest property didn't cause the project to attempt to be packaged. Rather than spend time experimenting, as it's already in the Unit project, I just added it to the infrastructure one too.

File: src/System.Private.ServiceModel/tests/Common/Infrastructure/Infrastructure.Common.csproj (hunk 1 above)
Comment (HongGit) [reply src/System.Private.ServiceModel/tests/Common/Infrastructure/Infrastructure.Common.csproj:6]: Huh, IsTest is removed, but add IsPackable

**Pull request merged**